pkg/TIPA.py

```
@@ -164,587 +165,498 @@ def __init__(self, img, mouse_pos, display_info_init, gui_queue):
-                if len(questchecker) == 1:
-                    lists = gp_soup.find("div", {"class": "mw-parser-output"}).findAll("ul")
-                    for child in lists:
-                        if child.find("font", {"color": "red"}):
-                            for item in child.findChildren():
-                                if item.getText()[0].isdigit():
-                                    questsListText.append(item.getText().strip())
-
-                    quests = "\n".join(questsListText)
+        except Exception as e:
+            if self.debug_mode >= 1:
+                logging.error("Unexpected error: ", e, exc_info=True)
```

Ruff (0.8.0)

Logging `.exception(...)` should be used instead of `.error(..., exc_info=True)`

(G201)